Bottom: 157f057fc8982c458959738c2a2c8d76e6c12c7e
Top:    f6c126b4dc97b984cc91ac7d62ba41848149e9bc
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-31 12:34:18 -0300

Refresh of manual-document-mt-safety-filesys.patch

---

diff --git a/manual/filesys.texi b/manual/filesys.texi
index 182fd33..cd23ff6d3 100644
--- a/manual/filesys.texi
+++ b/manual/filesys.texi
@@ -749,7 +749,7 @@ are very helpful for this purpose.
 @comment dirent.h
 @comment BSD/SVID
 @deftypefun int alphasort (const void *@var{a}, const void *@var{b})
-@safety{@mtsafe{}@asunsafe{glocale-revisit}@acunsafe{glocale-revisit}}
+@safety{@mtsafe{glocale}@asunsafe{asmalloc}@acsafe{memleak}}
 @c Calls strcoll.
 The @code{alphasort} function behaves like the @code{strcoll} function
 (@pxref{String/Array Comparison}).  The difference is that the arguments
@@ -763,8 +763,8 @@ than zero depending on the order of the two entries @var{a} and @var{b}.
 @comment dirent.h
 @comment GNU
 @deftypefun int versionsort (const void *@var{a}, const void *@var{b})
-@safety{@mtsafe{}@asunsafe{glocale}@acunsafe{glocale}}
-@c Calls strverscmp.
+@safety{@mtsafe{glocale}@assafe{}@acsafe{}}
+@c Calls strverscmp, which will evaluate .
 The @code{versionsort} function is like @code{alphasort} except that it
 uses the @code{strverscmp} function internally.
 @end deftypefun
@@ -797,7 +797,7 @@ argument.  Instead we provide the two replacement functions below.
 @comment dirent.h
 @comment GNU
 @deftypefun int alphasort64 (const void *@var{a}, const void *@var{b})
-@safety{@mtsafe{}@asunsafe{glocale-revisit}@acunsafe{glocale-revisit}}
+@safety{@mtsafe{glocale}@asunsafe{asmalloc}@acsafe{memleak}}
 @c See alphasort.
 The @code{alphasort64} function behaves like the @code{strcoll} function
 (@pxref{String/Array Comparison}).  The difference is that the arguments
@@ -811,7 +811,7 @@ than zero depending on the order of the two entries @var{a} and @var{b}.
 @comment dirent.h
 @comment GNU
 @deftypefun int versionsort64 (const void *@var{a}, const void *@var{b})
-@safety{@mtsafe{}@asunsafe{glocale}@acunsafe{glocale}}
+@safety{@mtsafe{glocale}@assafe{}@acsafe{}}
 @c See versionsort.
 The @code{versionsort64} function is like @code{alphasort64}, excepted that it
 uses the @code{strverscmp} function internally.
